### DISCUSSION TITLE
MultiFlow Chip test Does not seem to change flow rate

### DISCUSSION THREAD

**Discussion by schulmkg** [5axes/Calibration-Shapes#84] [Q&A]
So I printed both the Flow Tower and the multi flow chip test.  And the tower kinda got me close

but the multi chip test seemed to do nothing.

Running the GCode through Visual Studio Code and comparing the Gcode for each chip.  After slicing in Cura, the only differences were the start locations of the chip layers... AND the comment about how this next layer was going to be a different flow rate

Yet ALL the flow rates were F1200
ALL OF THEM

What am I doing wrong
cuz I really need to dial in flow rate as Im doing some airfoils and I am overextruding which means LOTS of sanding. which I hate

**Comment by 5axes** [comment]
Did you specify and set correctly the FlowTower script ? https://github.com/5axes/Calibration-Shapes/wiki/Postprocessing-scripts#flowtowerpy
